.travis.yml

```diff
@@ -26,8 +26,10 @@ before_script:
   - "export EVSS_SAMPLE_CLAIMANT_USER='{\"uuid\": \"1234\", \"first_name\": \"Jane\",\"last_name\": \"Doe\",\"edipi\": \"1105051936\",\"participant_id\": \"123456789\"}'"
   - export MVI_URL='http://ps-dev.commserv.healthevet.va.gov:8110/psim_webservice/IdMWebService'
   - export EVSS_S3_UPLOADS=false
-  - export VHA_MAPSERVER_URL='https://maps.va.gov/server/rest/services/PROJECTS/Facility_Locator/MapServer'
+  - export VHA_MAPSERVER_URL='https://services3.arcgis.com/aqgBd3l68G8hEFFE/ArcGIS/rest/services/VHA_Facilities/FeatureServer'
   - export VHA_MAPSERVER_LAYER='0'
+  - export NCA_MAPSERVER_URL='https://services3.arcgis.com/aqgBd3l68G8hEFFE/ArcGIS/rest/services/NCA_Facilities/FeatureServer'
```

Comment:
This is a nitpick and possibly annoying fix because it would require devops changes as well most likely, but how about just putting the host in an environment variable and then building the rest of the URI from within the class that requires it?